easter-egg-hunt.mdx

```diff
@@ -39,3 +39,18 @@ Stay tuned for daily hints on here, including our [launch page](https://xata.io/
 Once you find the hidden egg, click to reveal the riddle to solve.
 
 All easter eggs will still be available to retrospectively find if you start the hunt after day 1 but they will all be removed when the egg hunt closes, so don’t miss your chance to enter. Good luck 🍀
+
+**Announcing the winners:**
+We have randomly selected the winners who successfully found all of the daily hidden eggs and correctly solved the riddle. You will all be enjoying some time in a virtual reality with your new Meta Quest 3 VR headset.
+
+Congratulations to our 5 lucky winners:
+
+- [Aadarsh Kannan](https://twitter.com/dotAadarsh)
+- [Angus Dippenaar](http://@AngusDippenaar)
```

Comment:
This link is not correct.

Reply:
Yeah, I think that's the right person. There's another link to their twitter in the same draft doc. I'll update. Nice catch Elizabet!